css/at-rules/media.json

```diff
@@ -596,16 +596,22 @@
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/forced-colors",
             "support": {
               "chrome": {
-                "version_added": false,
+                "version_added": "79",
+                "flags": [
```

Comment:
Thank you very much for the update (and bringing in additional reviewers). One request here, could you duplicate this data in chrome_android below? I checked and this pref appears to be supported on both desktop and Android. Thanks again!

Reply:
Hi @melanierichards, do you plan to return to this PR? If not, let me know—we can take this over

Reply:
Sorry @ddbeck, got pulled into some other tasks during the feedback loop!

I think it might be better not to add chrome_android. Though the flag for forced color modes is technically supported on all platforms, it wouldn't be meaningful on platforms which don't have a forced color mode (like Windows High Contrast) to trigger the media query. Might be confusing to authors if we list it as supported behind a flag.

Reply:
Understood. Thank you for coming back to this, @melanierichards. I'll merge this shortly! 🎉

Reply:
Thank you!! And thanks for your patience :)